### PR TITLE
Add README and DEVELOPMENT instructions

### LINKED ISSUE #125: This repo needs a DEVELOPMENT.md
How do I test changes here (perhaps in concert with docs) locally (or remotely!)?

/assign @samodell

### PR DESCRIPTION
**Pull request by evankanderson**
Fixes #125 

I noticed that these were missing, and figured we might as well put some words here about how to manage this stuff.

I'm working to make the development loop a bit more straightforward, but the MacOS limits are real and really annoying.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Knative Website
+
+This repo contains the [hugo](https://gohugo.io) templates and themes for the
+Knative website. Basically, this should contain the formatting for the website,
+and [`knative/docs`](https://github.com/knative/docs) should contain the
+content for the website.
```
Comment (mattmoor) [inline README.md:6]: Format markdown:
```suggestion
and [`knative/docs`](https://github.com/knative/docs) should contain the content
for the website.
```

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+   [PostCSS](https://www.docsy.dev/docs/getting-started/#install-postcss)
+
+## Run locally
+
+You should be able to run `./scripts/localbuild.sh` and see your changes to the
+website and docs repo reflected. Note that the script **copies** the data from
+`../docs`, so you'll need to re-run the script to get fresh docs changes to show
+up on the local site.
+
+> @evankanderson is working on fixing this
```
Comment (mattmoor) [inline DEVELOPMENT.md:47]: Link to issue instead?

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+## On a Moc
```
Comment (mattmoor) [inline DEVELOPMENT.md:49]:
```suggestion
## On a Mac
```

File: DEVELOPMENT.md (hunk 2 above)
Comment (evankanderson) [reply DEVELOPMENT.md]: Done.

**Pull request merged**